src/platform/cocoamain.mm

```diff
@@ -1218,6 +1218,227 @@ - (IBAction)preferences:(id)sender {
     [NSApp stop:nil];
 }
 
+#ifdef HAVE_SPACEWARE
```

Comment:
Um. First you set SPACEWARE_FOUND to TRUE, then work around it in the build system to fix the linked files, then add this ifdef, which is always included. Why not just drop the ceremony entirely?